content-services/content-sources-backend
# Fixes 1971: Task info list filtering and new fields

## Summary
This adds 3 new fields to the task fetch and list API responses: type, repository name, repository (config) UUID. It also adds filtering by type and repository UUID to the task list endpoint.
 
## Testing steps
1. Create a repository with snapshot enabled so that a couple of tasks are produced.
2. List the tasks without filtering, filtering by repository UUID, and filtering by name. Check for new fields.
3. Fetch a task and check for new fields
